mafia/data/raretiles.json

```diff
@@ -538,6 +548,7 @@
   { "minute": 6350, "row": 6, "column": 2 },
   { "minute": 6376, "row": 7, "column": 6 },
   { "minute": 6380, "row": 6, "column": 1 },
+  { "minute": 6301, "row": 8, "column": 8 },
```

Comment:
This is between 6380 and 6391.
You call it 6301.
Did you mean 6381?

Reply:
Its accurate but wrong location, I just copy/paste it from what the bot reports.